sabnzbd/cfg.py

```diff
@@ -85,6 +85,7 @@ def validate_server(value):
 queue_complete_pers = OptionBool("misc", "queue_complete_pers", False)
 bandwidth_perc = OptionNumber("misc", "bandwidth_perc", 100, 0, 100)
 refresh_rate = OptionNumber("misc", "refresh_rate", 0)
+interface_settings = OptionStr("misc", "interface_settings", "'{}'")
```

Comment:
Is the extra `'` needed?

Reply:
I don't know. They're there when it saves because there are two layers of stringifying so I added them just in case.

Reply:
When it saves to `sabnzbd.ini`? Because the ConfigParser adds those on its own.
Does it work without the `'`?

Reply:
The page loads just fine with no default but when new values are stored it is in the format `'{"dateFormat":"fromNow",`, so for normal usage it is there.